Week1/homework/js-exercises/ex10.js

```diff
@@ -0,0 +1,13 @@
+//Exercise 10: Compare arrays
+//Declare 2 variables hold an array
+const arry1 = ['one', 'two', 'three', 'four', 'five'];
+const arry2 = ['one', 'two', 'three', 'four', 'five', 'seven', 'eight'];
+//Find out how to get the length of each array
+console.log(arry1.length);
+console.log(arry2.length);
+//Write a conditional statement
+if (arry1.length == arry2.length) {
```

Comment:
Ah, I see you used `==` here, should it not be`===` ?
You can read [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators) for clarification on `==` vs `===`:)

Reply:
i studied the difference between them, the two will work here, but i need to stuck with this  === more in future, when i read equal == come in to my mind, i will work on that , changes is done locally.